cfgov/jinja2/v1/_includes/organisms/filterable-list.html

```diff
@@ -172,11 +168,12 @@
                                         'alt': post.preview_image.alt if post.preview_image.alt else '',
                                     } if post.preview_image else {},
                                     'heading': heading,
-                                    'sub_heading': sub_heading,
                                     'body': post.preview_description,
                                     'links': links or null,
-                                    'link_image_and_heading': value.link_image_and_heading
-                                } ) }}
+                                    },
```

Comment:
I can't suggest this on the actual code because the first line isn't close enough to the current changes, but could you move the opening curly brace of the first param to the `info_unit()` macro onto its own line and indent its children to avoid the weirdness on these two lines?

Reply:
Fixed in 9878c52.